database/src/main/scala/cromwell/database/SqlDatabase.scala

```diff
@@ -180,6 +180,10 @@ trait SqlDatabase extends AutoCloseable {
                                     attempt: Int)
                                    (implicit ec: ExecutionContext): Future[Seq[Metadatum]]
 
+  protected def queryMetadataEventsWithKeyLike(workflowUuid: String,
```

Comment:
I'll add in that I found the `KeyLike` thing unwieldy but not enough to chime in. Since it has been brought up by @cjllanwarne I'll add a +0.5